src/sentry/tasks/post_process.py

```diff
@@ -249,18 +270,25 @@ def handle_group_owners(project, group, owners, owner_source):
         pass
 
 
-def update_existing_attachments(event):
+def update_existing_attachments(job):
```

Comment:
This doesn't look like its updating anything in `job` - maybe rename this to `process_existing_attachments`?

Reply:
I didn't write this one so tried to touch as little as I could. Happy to rename it though